packages/dialog/README.md

```diff
@@ -377,6 +377,45 @@ of the shadow root of `mwc-dialog` or the
 [root node](https://developer.mozilla.org/en-US/docs/Web/API/Node/getRootNode)
 of the dialog.
 
+### Form Handling
```

Comment:
Can you make this more generic than "form handling". Something along the lines of conditionally closing dialog. This is especially since we're still working on how normal mwc elements will worl with forms.